src/Sql/Sql.sqlproj

```diff
@@ -309,5 +309,6 @@
     <Build Include="dbo\Stored Procedures\TaxRate_Archive.sql" />
     <Build Include="dbo\Stored Procedures\OrganizationUser_ReadByOrganizationIdEmail.sql" />
     <Build Include="dbo\Stored Procedures\OrganizationUserOrganizationDetails_ReadByUserIdStatusOrganizationId.sql" />
+    <Build Include="dbo\quartz.sql" />
```

Comment:
Let's move this out of here and into a separate database, this could/would be an administrative database in our cloud infrastructure. We would leave this out of our source repository